testutils/src/main/scala/za/co/absa/enceladus/testutils/models/ProcessMeasurement.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.models
+
+case class ProcessMeasurement[T](runTime: Long, returnValue: T)
```

Comment:
Not sure if `ProcessMeasurement` is a very descriptive name. So far I cannot find a good replacement. Only something quite long, such as `OutputWithElapsedTime`. Maybe you could think of something...

Reply:
Or... maybe use a `Pair` instead of a case class for that time measurement function... not sure what's better.